README.md

```diff
@@ -5,74 +5,86 @@
 [The USAspending website](https://www.usaspending.gov/) is the public-facing site offering information on Government spending for the United States. It utilizes the [Data Transparency User Interface Library](https://github.com/fedspendingtransparency/data-transparency-ui).
 
 ## Development Set Up
-To run the USAspending website locally, follow the directions below.
+- To contribute to frontend development, follow _Install and Configure_ below. 
+- To simply run the USAspending website locally, jump to _Build and Run with Docker_ below.
 
-Assumptions:
+_Assumptions_:
 
 * You're able to install software on your machine.
 * You have git installed on your machine and are able to clone code repositories from GitHub. If this isn't the case, the easiest way to get started is to install [GitHub Desktop](https://desktop.github.com/ "GitHub desktop"), available for Windows or Mac.
 * You're familiar with opening a terminal on your machine and using the command line as needed.
 
-### Install Prerequisites and Code
-
-1. If you're not already running Node.js, download and run the installer for your operating system. We build using **Node.js 10.15.3 (LTS)** which we recommend downloading using [Node Version Manager (nvm)](https://github.com/nvm-sh/nvm).
-
-2. You also need npm. We use version `6.14.5`.
-
-```shell
-        npm i -g npm@6.14.5
-```
-
-3. From the command line, clone the USAspending website repository from GitHub to your local machine:
-
-```shell
-        git clone https://github.com/fedspendingtransparency/usaspending-website.git
-```
-
-4. Change to the `usaspending-website` directory that was created when you cloned the USAspending Website repository.
-
-```shell
-        cd usaspending-website
-```
-
-5. Perform a clean install to get an exact dependency tree:
-
-```shell
-        npm ci
-```
-
-### Configuration File
-
-The USAspending website expects a certain number of environment variables to be defined at runtime. The important ones are listed below along with their default values:
-
-| ENV VAR        | VALUE                            |
-|----------------|----------------------------------|
-| USASPENDING_API| https://api.usaspending.gov/api/ |
-| MAPBOX_TOKEN   | ''                               |
-| GA_TRACKING_ID | ''                               |
+### Install and Configure
+_Get the code and install the runtime and dependencies_
+
+1. From the command line, clone the USAspending website repository from GitHub to your local machine, and get into the root directory:
+    ```shell
+    $ git clone https://github.com/fedspendingtransparency/usaspending-website
+    $ cd usaspending-website
+    ```
+1. Download [Node Version Manager (`nvm`)](https://github.com/nvm-sh/nvm) and install **Node.js `12.18.4`**
+    ```shell
+    $ nvm install 12.18.4
+    ```
+1. Set Node Package Manager (`npm`) CLI to version `6.14.5`.
```

Comment:
Node install comes with npm. At one point we needed to install a higher version of npm due the node version we were running but since we moved to 12 this step is not needed and we can delete this.